src/index.ts

```diff
@@ -95,6 +96,56 @@ export const createInstance = (config: Config): Instance => {
 
       return tx;
     },
+    lockDomainMetadata: async (
+      domainId: string,
+      lockStatus: boolean,
+      signer: ethers.Signer
+    ): Promise<ethers.ContractTransaction> => {
+      const registrar: Registrar = await getRegistrar(signer, config.registrar);
+      const potentialOwner = await signer.getAddress();
+
+      const tx = await actions.lockDomainMetadata(
+        domainId,
+        lockStatus,
+        potentialOwner,
```

Comment:
Passing `potentialOwner` isn't actually required here since you can get the signer and do the `.getAddress()` call from within these methods.